packages/web/src/components/Plan.tsx

```diff
@@ -3,38 +3,12 @@ import { Button, Card, Col, Empty, Icon, Row, Tooltip, Typography } from "antd";
 import * as _ from "lodash";
 import QueueAnim from "rc-queue-anim";
 import React from "react";
-import {
-    DragDropContext,
-    Draggable,
-    DraggableProvided,
-    Droppable,
-    DroppableProvided,
-    DropResult,
-    ResponderProvided,
-} from "react-beautiful-dnd";
+import { DragDropContext, Draggable, DraggableProvided, Droppable, DroppableProvided, DropResult, ResponderProvided } from "react-beautiful-dnd";
```

Comment:
Were these auto formatted btw?  vscode, assuming you have the `prettier` extension installed should keep it formatted like before this change.

Reply:
hmm that's weird yea they're autoformatted on save even though I have prettier installed lemme look into this